scripts/portstat

```diff
@@ -31,289 +30,180 @@ NStats = namedtuple("NStats", "rx_ok, rx_err, rx_drop, rx_ovr, tx_ok,\
 header = ['Iface', 'RX_OK', 'RX_RATE', 'RX_UTIL', 'RX_ERR', 'RX_DRP', 'RX_OVR',
           'TX_OK', 'TX_RATE', 'TX_UTIL', 'TX_ERR', 'TX_DRP', 'TX_OVR']
 
-port_intf_dict = {
-    "xe0": "Ethernet0",
-    "xe1": "Ethernet4",
-    "xe2": "Ethernet8",
-    "xe3": "Ethernet12",
-    "xe4": "Ethernet16",
-    "xe5": "Ethernet20",
-    "xe6": "Ethernet24",
-    "xe7": "Ethernet28",
-    "xe8": "Ethernet32",
-    "xe9": "Ethernet36",
-    "xe10": "Ethernet40",
-    "xe11": "Ethernet44",
-    "xe12": "Ethernet48",
-    "xe13": "Ethernet52",
-    "xe14": "Ethernet56",
-    "xe15": "Ethernet60",
-    "xe16": "Ethernet64",
-    "xe17": "Ethernet68",
-    "xe18": "Ethernet72",
-    "xe19": "Ethernet76",
-    "xe20": "Ethernet80",
-    "xe21": "Ethernet84",
-    "xe22": "Ethernet88",
-    "xe23": "Ethernet92",
-    "xe24": "Ethernet96",
-    "xe25": "Ethernet100",
-    "xe26": "Ethernet104",
-    "xe27": "Ethernet108",
-    "xe28": "Ethernet112",
-    "xe29": "Ethernet116",
-    "xe30": "Ethernet120",
-    "xe31": "Ethernet124"
+counter_bucket_dict = {
+    'SAI_PORT_STAT_IF_IN_UCAST_PKTS': 0,
+    'SAI_PORT_STAT_IF_IN_NON_UCAST_PKTS': 0,
+    'SAI_PORT_STAT_IF_IN_ERRORS': 1,
+    'SAI_PORT_STAT_IF_IN_DISCARDS': 2,
+    'SAI_PORT_STAT_ETHER_TX_OVERSIZE_PKTS': 3,
+    'SAI_PORT_STAT_IF_OUT_UCAST_PKTS': 4,
+    'SAI_PORT_STAT_IF_OUT_NON_UCAST_PKTS': 4,
+    'SAI_PORT_STAT_IF_OUT_ERRORS': 5,
+    'SAI_PORT_STAT_IF_OUT_DISCARDS': 6,
```

Comment:
do we know if OUT_DISCARDS map to all the counters we used to get from bshell? TDBGC0, 3, 5, 8 and the UCQ_DROP_PKT counters?